config/prometheus/monitor.yaml

```diff
@@ -18,3 +18,32 @@ spec:
   selector:
     matchLabels:
       control-plane: controller-manager
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+      - endpoints
+      - pods
+    verbs:
+      - get
+      - list
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+  - kind: ServiceAccount
+    name: prometheus-k8s
+    namespace: openshift-monitoring
```

Comment:
This is OpenShift-specific - should we wait until we have a downstream repo before we add this file?

Reply:
i added it right now, so that we won't forget it ( also to allow checking on OCP). in ant case, the prometheus is commented in the default kustomezation, so it is not deployed on the K8s